tardis/model/radiation_field_state.py

```diff
@@ -44,4 +46,6 @@ def calculate_mean_intensity(self, nu):
         intensity : u.Quantity
             Intensity of the radiation field at the given frequency
         """
-        return self.dilution_factor * intensity_black_body(nu, self.t_radiative)
+        return self.dilution_factor * intensity_black_body(
+            nu[np.newaxis].T, self.t_radiative
```

Comment:
Why is this transpose happening here?

Reply:
this is the number of nues vs the number of shells - what comes out is a 2D frame